backend/amd64/reg_class.ml

```diff
@@ -48,16 +47,40 @@ module T = struct
          "%xmm8"; "%xmm9"; "%xmm10"; "%xmm11";
          "%xmm12"; "%xmm13"; "%xmm14"; "%xmm15" |]
 
+  let[@ocamlformat "disable"] ymm_name =
+    match Config.ccomp_type with
+    | "msvc" ->
+      [| "ymm0"; "ymm1"; "ymm2"; "ymm3"; "ymm4"; "ymm5"; "ymm6"; "ymm7";
```

Comment:
(It was already a bit unpleasant
with only `xmm`, and having it
three times makes it worse.
Could we avoid these array
literals? - quite possibly in another
pull request.)

Reply:
Fixed